app/admin.go

```diff
@@ -112,7 +183,63 @@ func (s *Server) GetLogsSkipSend(page, perPage int) ([]string, *model.AppError)
 					if err != nil {
 						return nil, model.NewAppError("getLogs", "api.admin.file_read_error", nil, "", http.StatusInternalServerError).Wrap(err)
 					}
-					lines = append(lines, string(line))
+
+					filtered := false
+
+					logLevels := logFilter.LogLevels
+					if len(logLevels) > 0 {
```

Comment:
Filtering can be done in separate function, also I'd like to have a way that system admin will get notified properly if there are errors during filtering. It seems like we are just logging in debug level which sometimes (if they set log level above debug) is basically piping to `/dev/null`.